ddtrace/opentelemetry/span.go

```diff
@@ -35,6 +38,7 @@ type span struct {
 	finishOpts []tracer.FinishOption
 	statusInfo
 	*oteltracer
+	events []otelsdk.Event
```

Comment:
I'm wondering if the `events` field should contain a slice of `ddSpanEvent`s instead of `otelsdk.Event`s

Reply:
I'd use a slice of `ddSpanEvent`s instead of `otelsdk.Event`s.